tsconfig.json

```diff
@@ -0,0 +1,48 @@
+{
+  "compileOnSave": true,
+  "compilerOptions": {
+    "stripInternal": true,
+
+    // Enables all strict type checking options.
+    "strict": true,
+
+    // Library files to be included in the compilation. Basically which "core
+    // language features" TypeScript should enable.
+    "lib": ["es2015", "es2016", "es2017"],
+
+    // Which version of EcmaScript TypeScript should transpile to. Because of
+    // how Babel and Webpack is set up we currently target `esnext`, and let
+    // Babel take care of transpiling it down.
+    "target": "esnext",
+
+    // Specifies module code generation.
+    "module": "esnext",
+
+    // Allows default imports from modules with no default export. This does not affect code emit, just type checking.
+    "allowSyntheticDefaultImports": true,
```

Comment:
I think we can remove this, as it should be handled automatically by the `esModuleInterop` flag.

Reply:
It should, but it doesn't. Let me investigate, maybe I'm doing something wrong.

Reply:
Here is what I see in TS source code:
```ts
function getAllowSyntheticDefaultImports(compilerOptions) {
        var moduleKind = getEmitModuleKind(compilerOptions);
        return compilerOptions.allowSyntheticDefaultImports !== undefined
            ? compilerOptions.allowSyntheticDefaultImports
            : compilerOptions.esModuleInterop
                ? moduleKind !== ts.ModuleKind.None && moduleKind < ts.ModuleKind.ES2015
                : moduleKind === ts.ModuleKind.System;
    }
```

Reply:
Ah, fun. Okey, let's keep it. Want to add a note about that in the comment?